tests/test.rs

```diff
@@ -456,7 +456,7 @@ fn test_integration_zip() {
     for compiler in compilers {
         let is_llvm = compiler == "clang++";
         let name = if is_llvm { "llvm" } else { "gcc" };
-        let path = &PathBuf::from("tests/basic_zip");
+        let path = &PathBuf::from("tests/basic_zip_zip");
```

Comment:
we should update https://github.com/mozilla/grcov/pull/150/files#diff-dcd61c41ae35cab6b9646368cca02e49R391 to ignore this too.